forge-gui/res/cardsfolder/upcoming/jedit_ojanen_mercenary.txt

```diff
@@ -0,0 +1,8 @@
+Name:Jedit Ojanen, Mercenary
+ManaCost:1 W U
+Types:Legendary Creature Cat Mercenary
+PT:3/3
+T:Mode$ ChangesZone | ValidCard$ Creature.Legendary+YouCtrl | Origin$ Any | Destination$ Battlefield | TriggerZones$ Battlefield | Execute$ TrigToken | OptionalDecider$ You | TriggerDescription$ Whenever CARDNAME or another legendary creature enters the battlefield, you may pay {G}. If you do, create a 2/2 green Cat Warrior creature token with forestwalk.
```

Comment:
ValidCard$ should include Card.Self, if for some reason it doesn't ETB as creature